enterprise/api/v1/views.py

```diff
@@ -1249,3 +1250,39 @@ def post(self, request):
                 {'error': str('Notification read request failed')},
                 status=HTTP_500_INTERNAL_SERVER_ERROR
             )
+
+
+class EnterpriseCustomerReportTypesView(APIView):
+    """
+    API for getting the report types associated with an enterprise customer
+    """
+    authentication_classes = [SessionAuthentication]
+    permission_classes = [permissions.IsAuthenticated, permissions.IsAdminUser]
+    http_method_names = ['get']
+
+    @permission_required(
+        'enterprise.can_access_admin_dashboard',
+        fn=lambda request, enterprise_uuid: enterprise_uuid
+    )
+    def get(self, request, enterprise_uuid):
+        """
+        Get the dropdown choices for EnterpriseCustomerReportingConfiguration
+        """
+        enterprise_customer = get_enterprise_customer(enterprise_uuid)
+        if not enterprise_customer:
+            return Response({"detail": "Could not find the enterprise customer."}, status=HTTP_404_NOT_FOUND)
+
+        meta = models.EnterpriseCustomerReportingConfiguration._meta
+        choices = {}
+        for field in meta.get_fields():
+            if hasattr(field, 'choices'):
```

Comment:
to remove empty lists from the response we can update this condition
`if hasattr(field, 'choices') and field.choices`